### ISSUE TITLE
Load czi file is not deterministic

### ISSUE THREAD

**Issue by WinterPan2017** [cgohlke/czifile#7]
Thank you release the tool. We found that sometimes using czifile package to load .czi file is not deterministic.  

```
czi = czifile.CziFile(file_path)
meta = czi.metadata(raw=False)
channel_info = meta['ImageDocument']['Metadata']['DisplaySetting']['Channels']['Channel']
if type(channel_info) == list:
    channel_names = [c['Name'] for c in channel_info]
    channel_index = channel_names.index('EGFP')
else:
    channel_index = 0
dim_indices = [':'] * len(czi.axes)
dim_indices[czi.axes.index('C')] = 'channel_index'
image = eval('czi.asarray(resize=False)[%s].squeeze()' % ','.join(dim_indices))
```

We run the same code (above) twice and get different arrays. It seems a bug in this package. We recommend using [aicspylibczi](https://pypi.org/project/aicspylibczi/) to avoid the problem.

**Comment by cgohlke** [comment]
That's a know issue with files containing subresolution levels. Czifile is not aware of levels. You'll have to filter/sort segments yourself.

**Comment by cgohlke** [comment]
Should be fixed in v2026.3.12.
